submissions/Oleksandr-Heleta/task_js_oop/app.js

```diff
@@ -0,0 +1,131 @@
+const START_X = 202,
+    START_Y = 404,
+    CELL_WIDTH = 100,
+    CELL_HEIGTH = 83,
+    ROW_POSITION = [60, 143, 228],
```

Comment:
As a peer developer I am tasked to change the playfield params. I need to add more rows. Same questions as above follow.

Ideally i'd like to add row number(s).